images/installer/bin/assemble

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+#
+# Assemble script for the openshift/openshift-ansible image.
+# The 'assemble' script builds the application source so that it is ready to run.
+#
+# For more information refer to the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+
+if [[ "$1" == "-h" ]]; then
+	# If the assemble script is executed with '-h' flag,
+	# print the usage.
+	exec /usr/bin/usage
+fi
+
+# Restore artifacts from the previous build (if they exist).
+#
+if [ "$(ls /tmp/artifacts/ 2>/dev/null)" ]; then
+  echo "---> Restoring build artifacts..."
+  mv /tmp/artifacts/. ./
+fi
+
+echo "---> Installing application source..."
+cp -Rf /tmp/src/. ./
+
+echo "---> Building application from source..."
+# pre-create PYTHONUSERBASE/lib/*, otherwise pip creates it with mode 0700
+# which prevents containers with arbitrary UIDs to access local packages
+mkdir -p ${HOME}/.local/lib/python2.7/site-packages
+if [[ -v PYTHON_REQUIREMENTS ]]; then
+  pip install --no-cache-dir -r ${PYTHON_REQUIREMENTS}
+elif [[ -e requirements.txt ]]; then
+  pip install --no-cache-dir --user -r requirements.txt
+fi
+
+if [[ -v INSTALL_OC ]]; then
+  pip install --no-cache-dir --user requests
+  echo "---> Installing 'oc' binary..."
+  TMPDIR=$(mktemp -d)
+  cd ${TMPDIR}
+  OC_BINARY_URL=$(python -c "import requests;releases = requests.get('https://api.github.com/repos/openshift/origin/releases').json();print [s for s in [r for r in releases if not r['prerelease'] and '1.4' in r['name']][0]['assets'] if 'linux-64' in s['browser_download_url']][0]['browser_download_url']")
```

Comment:
gets latest origin oc... origin-only of course. err, it's a little unclear to me what this actually ends up with... 1.4 releases?

downstream, RPM deps will pull in oc